src/kmerseek/index.py

```diff
@@ -62,13 +62,18 @@ def make_rocksdb_index(sig, moltype, ksize, scaled):
 @click.option("--moltype", default="hp")
 @click.option("--ksize", type=int, default=24)
 @click.option("--scaled", type=int, default=5)
+@click.option("--do-kmer-extraction", is_flag=True, default=False)
```

Comment:
If the default is to not run kmer extraction, I would suggest using `--extract-kmers` as the option name.

If the default is to run kmer extraction, I would suggest using `--skip-extract-kmers` as the option name.